### PR TITLE
Keyboard integration for streamlink quality confirmation

### PR DESCRIPTION
**Pull request by M4tthewDE**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

This pr allows the confirmation of the streamlink quality selection by hitting Enter, without having to tab to the "Ok"-Button. 

Not experienced with qt, so feedback appreciated :)

### CODE REVIEW

File: src/widgets/dialogs/QualityPopup.cpp [hunk 1]
```diff
@@ -46,6 +46,18 @@ void QualityPopup::showDialog(const QString &channelName, QStringList options)
...
+{
+    if (event->key() == Qt::Key_Enter || event->key() == Qt::Key_Return)
+    {
+        this->okButtonClicked();
+    }
+    else if (event->key() == Qt::Key_Escape)
+    {
+        this->cancelButtonClicked();
+    }
+}
```
Comment (Mm2PL) [inline src/widgets/dialogs/QualityPopup.cpp:59]: Use `createWindowShortcut` instead

**Review by Mm2PL — CHANGES_REQUESTED** [review]

**Comment by zneix** [comment]
I think this is such a small thing that it doesn't necessarily require a changelog entry.

**Comment by zneix** [comment]
I think instead of adding a shortcut for Enter, rather changing the default button or something along these lines should be done instead. That'd be consistent with all other button behavior's among the codebase and cleaner in my opinion. I'll to fiddle around this one for a little bit now.
Just making a comment before pajlada gets to see this PR.

**Comment by pajlada** [comment]
Fixed in #3169

**Pull request closed without merging**